pubspec.yaml

```diff
@@ -83,11 +85,22 @@ dependency_overrides:
   webview_flutter: ^4.0.1
   fwfh_webview: 0.7.0
   geolocator: ^9.0.2
+  path: ^1.8.3
+  at_sync_ui_flutter:
+    git:
+      url: https://github.com/atsign-foundation/at_widgets.git
+      ref: fix/sync_snackbar_issue
+      path: packages/at_sync_ui_flutter
```

Comment:
Can we replace sync_ui reference to `at_sync_ui_flutter 1.0.8 ` ?

Reply:
Yes, I've deleted it and used at_sync_ui_flutter 1.0.8.

Reply:
I've pushed code. Please check it.